test/jdk/javax/swing/plaf/aqua/CustomComboBoxFocusTest.java

```diff
@@ -55,29 +55,24 @@
 import javax.swing.JTextField;
 import javax.swing.SwingUtilities;
 
-import jdk.test.lib.Platform;
-
 public class CustomComboBoxFocusTest {
 
     private static CustomComboBoxFocusTest test = null;
+    static int colorTolerance = 5;
 
-    public static void main(String[] args) {
-        if (!Platform.isOSX()) {
+    public static void main(String[] args) throws Exception {
+        if (!System.getProperty("os.name").toLowerCase().contains("os x")) {
```

Comment:
@prsadhuk Since the tag **@requires (os.family == "mac")** is added, is this check necessary or is it added for standalone testing purpose?

Reply:
Yes for standalone testing for which I also removed dependancy on jtreg Platform class

Reply:
Any reviewers please?

Reply:
I prefer .toLoweCase().startsWith("mac") since the OS is called "macOS" not "macOS X" but that's just me ..